ecosystem_tests/ecosystem_tests_cli/ecosystem_tests.py

```diff
@@ -138,7 +138,15 @@ def __init__(self):
         a developer sees an option. It it can receive arguments, it's a
         method - if not, it's an attribute.
         """
-
+        self.version = click.option('-V',
```

Comment:
This may confuse users, because `--version` and `-V` are the flags for the`python` binary for the version of Python. E.g.: 
```bash
➜  cloudify-ecosystem-test git:(RD-3293-aarch64) python -V
Python 3.7.10 (77787b8f4c49, May 19 2021, 05:28:43)
[PyPy 7.3.5 with GCC Apple LLVM 12.0.5 (clang-1205.0.22.9)]
```
That said, I don't have any good suggestions.